src/main/java/net/sf/jabref/gui/journals/AbbreviationsFile.java

```diff
@@ -0,0 +1,132 @@
+/*  Copyright (C) 2016 JabRef contributors.
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 2 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License along
+    with this program; if not, write to the Free Software Foundation, Inc.,
+    51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+package net.sf.jabref.gui.journals;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.io.OutputStreamWriter;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+import java.util.Objects;
+
+import javafx.beans.property.ReadOnlyBooleanProperty;
+import javafx.beans.property.SimpleBooleanProperty;
+import javafx.beans.property.SimpleListProperty;
+import javafx.collections.FXCollections;
+
+import net.sf.jabref.Globals;
+import net.sf.jabref.logic.journals.Abbreviation;
+import net.sf.jabref.logic.journals.JournalAbbreviationLoader;
+
+/**
+ * This class provides a model for abbreviation files.
+ * It actually doesn't save the files as objects but rather saves
+ * their paths. This also allows to specify pseudo files as placeholder objects.
+ */
+public class AbbreviationsFile {
+
+    private final SimpleListProperty<AbbreviationViewModel> abbreviations = new SimpleListProperty<>(
+            FXCollections.observableArrayList());
+    private final ReadOnlyBooleanProperty isPseudoFile;
```

Comment:
Maybe rename isPseudoFile -> isBuiltinList to better reflect what the actual purpose of this field is.